packages/wrangler/src/__tests__/init.test.ts

```diff
@@ -2719,17 +2719,18 @@ describe("init", () => {
 			DATA_BLOB_ONE = \\"DATA_BLOB_ONE\\"
 			DATA_BLOB_TWO = \\"DATA_BLOB_TWO\\"
 
-			[[unsafe.bindings]]
-			type = \\"some unsafe thing\\"
-			name = \\"UNSAFE_BINDING_ONE\\"
+			[unsafe]
```

Comment:
Just to double check, this isn't a breaking change for users of current unsafe.bindings right? They won't have to nest their entries under another `[unsafe]`?

Reply:
Doesn't look like it, judging by the config shape:
```
// before
unsafe: { bindings: config.unsafe?.bindings },

// after
unsafe: {
	bindings: config.unsafe?.bindings,
	metadata: config.unsafe?.metadata,
},
```

Reply:
I think that actually this TOML is just not correct given the config?

Reply:
The `[unsafe]` line is ignored by TOML.

Reply:
This block of TOML results in the following equivalent JSON, whether or not the `[unsafe]` line is there:

```json
{
    "unsafe": {
        "bindings": [
            {
                "type": "some unsafe thing",
                "name": "UNSAFE_BINDING_ONE",
                "data": {
                    "some": {
                        "unsafe": "thing"
                    }
                }
            },
            {
                "type": "another unsafe thing",
                "name": "UNSAFE_BINDING_TWO",
                "data": 1337
            }
        ]
    }
}
```

Reply:
Which is identical to what was here before.

Reply:
But I see that this is some weirdness in how we are converting the binding info from the dashboard back into TOML.

Reply:
This should continue to work as expected for people with existing bindings.
I tested this by publishing my worker which has some unsafe bindings and no metadata and everything went as expected.

Reply:
Yeah it is just a shame that the TOML stringifier creates this weird confusing wrangler.toml.